mozdownload/cli.py

```diff
@@ -0,0 +1,224 @@
+                        dest='password',
+                        metavar='PASSWORD',
+                        help='Password for basic HTTP authentication.')
+    parser.add_argument('--retry-attempts',
+                        dest='retry_attempts',
+                        default=0,
+                        type=int,
+                        metavar='RETRY_ATTEMPTS',
+                        help='Number of times the download will be attempted in '
+                             'the event of a failure, default: %(default)s')
+    parser.add_argument('--retry-delay',
+                        dest='retry_delay',
+                        default=10.,
+                        type=float,
+                        metavar='RETRY_DELAY',
+                        help='Amount of time (in seconds) to wait between retry '
+                             'attempts, default: %(default)s')
+    parser.add_argument('--timeout',
+                        dest='timeout',
+                        type=float,
+                        metavar='TIMEOUT',
+                        help='Amount of time (in seconds) until a download times out')
+    parser.add_argument('--log-level',
+                        action='store',
+                        dest='log_level',
+                        default='INFO',
+                        metavar='LOG_LEVEL',
+                        help='Threshold for log output (default: %(default)s)')
+
+    # Group for daily builds
+    group = parser.add_argument_group('Daily builds', 'Extra options for daily builds.')
+    group.add_argument('--branch',
+                       dest='branch',
+                       default='mozilla-central',
+                       metavar='BRANCH',
+                       help='Name of the branch, default: "%(default)s"')
+    group.add_argument('--build-id',
+                       dest='build_id',
+                       metavar='BUILD_ID',
+                       help='ID of the build to download')
+    group.add_argument('--date',
+                       dest='date',
+                       metavar='DATE',
+                       help='Date of the build, default: latest build')
+
+    # Group for tinderbox builds
+    group = parser.add_argument_group('Tinderbox builds', 'Extra options for tinderbox builds.')
+    group.add_argument('--debug-build',
+                       dest='debug_build',
+                       action='store_true',
+                       help='Download a debug build')
+
+    # Group for try builds
+    group = parser.add_argument_group('Try builds', 'Extra options for try builds.')
+    group.add_argument('--changeset',
+                       dest='changeset',
+                       help='Changeset of the try build to download')
+
+    args = parser.parse_args()
+
+    # Gives instructions to user when no arguments were passed
+    if len(sys.argv) == 1:
+        print __doc__
+        parser.format_usage()
+        print 'Specify --help for more information on args. ' \
```

Comment:
Well, without using rounded brackets for the print statement the next line would have to start at the same column without indentation. It's kinda hard to read. So I will change the file to use `print('...')` instead.